charts/topolvm/README.md

```diff
@@ -83,6 +83,7 @@ You need to configure kube-scheduler to use topolvm-scheduler extender by referr
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | cert-manager.enabled | bool | `false` | Install cert-manager together. # ref: https://cert-manager.io/docs/installation/kubernetes/#installing-with-helm |
+| controller.additionalLabels | object | `{}` | Additional labels to be set on the controller pod |
```

Comment:
Could you also add `lvmd.additionalLabels` and `scheduler.additionalLabels`? These parameters would be helpful for someone.